### PR TITLE
Add enums and add deletes for objects in seed file

### PR DESCRIPTION
**Pull request by ILyell**
## Description

This adds some enums to the back end if we need them and adds delete methods for the seed file for deployment.

## Checklist
- [x] I have reviewed the code changes.
- [x] I have tested this code.
- [x] I have updated the documentation.
- [x] I have checked for and resolved any merge conflicts.
- [x] I have assigned reviewers for this PR.

### CODE REVIEW

File: app/models/answered_question.rb [hunk 1]
```diff
@@ -5,4 +5,6 @@ class AnsweredQuestion < ApplicationRecord
   validates :user_game_id, presence: true
   validates :question, presence: true
   validates :result, presence: true
+
+  enum result: [ :incorrect, :correct, :self_correct ]
```
Comment (ethanrossblack) [inline app/models/answered_question.rb:9]: need to add "skipped"

File: config/schedule.rb [hunk 2]
```diff
@@ -22,6 +22,6 @@
 # set :bundle_command, "/Users/ianlyell/.rbenv/shims/bundle exec"
 
 #Creates a new daily game in the DB at 1:00 AM everyday.
-every 1.minute do
+every 1.hour do
```
Comment (topher-nullset) [inline config/schedule.rb:25]: But I like spam.

**Review by topher-nullset — APPROVED** [review]
LGTM

File: app/models/question.rb [hunk 3]
```diff
@@ -2,4 +2,6 @@ class Question < ApplicationRecord
   has_many :game_questions
   has_many :games, through: :game_questions
   validates :clue, :answer, :category, :clue_id, :value, :category_id, presence: true
+
+  enum status: [:valid_quesiton, :invalid_quesiton]
```
Comment (ethanrossblack) [inline app/models/question.rb:6]: What makes a question valid or invalid?

Is this associated with the "value" column?

**Pull request merged**